### ISSUE TITLE
Zeus Interface Issue

### ISSUE THREAD

**Issue by ryanparsons7** [Sniperhid/1tac_admin#11]
When I give myself or another player Zeus, when using the Zeus interface it creates duplicate option/properties dialog boxes when placing modules, editing units etc.

This does not happen when accessing Zeus via a unit that was given Zeus at the start using the  "Game Master" module in eden.

Only occurs on dedicated server

Heres a video below showing the issue:
https://www.youtube.com/watch?v=w2lJEmzRXAg

**Comment by Sniperhid** [comment]
Good catch and confirmed. Only happens seems to happen in MP. Not quite sure how to prevent this =/
